pages/index.vue

```diff
@@ -85,6 +86,9 @@ export default Vue.extend({
     // fixes https://hauptstadt.atlassian.net/browse/HA-121
     await this.$vuetify.goTo(0, { duration: 0 })
 
+    const loginBypass = new LoginBypass(this.$cookies)
+    loginBypass.enable(this.$route.query.key as string)
```

Comment:
We could probably remove the key from the query after enabling the bypass so that it's a little bit less obvious / easy to bookmark the url. Otherwise: nice and clean code.